workloads/kube-burner/common.sh

```diff
@@ -70,20 +70,19 @@ run_workload() {
   CMD="timeout ${JOB_TIMEOUT} ${KUBE_DIR}/kube-burner init --uuid=${UUID} -c $(basename ${WORKLOAD_TEMPLATE}) --log-level=${LOG_LEVEL}"
   # When metrics or alerting are enabled we have to pass the prometheus URL to the cmd
   if [[ ${INDEXING} == "true" ]] || [[ ${PLATFORM_ALERTS} == "true" ]] ; then
-    if [[ -n ${PROM_URL} ]] && [[ ${PROM_URL} != "None" ]]; then
-      CMD+=" -u ${PROM_URL}"
-    fi
-    CMD+=" -t ${PROM_TOKEN}"
+      if [[ -n ${METRICS_ENDPOINT} ]]; then
```

Comment:
Can you add this var to the docs?

Reply:
Its already present in the README.md: https://github.com/cloud-bulldozer/e2e-benchmarking/pull/537/files#diff-1bcb772094813e10febb0bc183aa8270d15743e10caceebb2b6fdfc829e6c54eR33. Let me know if there are additional places where this var needs to be added. Thanks!